tests/wdio.theia-https.conf.js

```diff
@@ -0,0 +1,31 @@
+const wdio = require('./wdio.base.conf.js');
+const headless = true;
+
+const config = wdio.makeConfig(headless);
+
+const cliPortKey = '--theia-port';
+const cliPortIndex = process.argv.indexOf(cliPortKey);
+const masterPort = cliPortIndex > -1 ? process.argv[cliPortIndex + 1] : 0; // 0 if master
+if (typeof masterPort === 'undefined') {
+    throw new Error(`${cliPortKey} expects a number as following argument`);
+}
+
+/*  The tests call the url / so we are not able to use the token. 
```

Comment:
Do we still want to keep this comment or is it leftover?

Reply:
I'd prefer to keep it, in order to include a future check for using tokens.

Reply:
No problem it's up to you :)